app/jobs/zendesk_tickets_job.rb

```diff
@@ -0,0 +1,23 @@
+class ZendeskTicketsJob < ActiveJob::Base
+  queue_as :zendesk
+
+  def perform(feedback)
+    ZendeskAPI::Ticket.create!(
+      Rails.configuration.zendesk_client,
+      description: feedback.body,
+      requester: {
+        email: feedback.email_address,
+        name: 'Unknown'
+      },
+      custom_fields: custom_fields(feedback)
+    )
+  end
+
+  def custom_fields(feedback)
+    [
+      { id: '23730083', value: feedback.referrer },
+      { id: '23757677', value: 'prison_visits' },
+      { id: '23791776', value: feedback.user_agent }
```

Comment:
According to the [Zendesk API docs on creating tickets](https://developer.zendesk.com/rest_api/docs/core/tickets#setting-custom-field-values), they relate to our custom ticket fields.

After poking around in the admin UI, it would appear that they are:

- `23730083` = `URL` text field
- `23757677` = `Service` dropdown (with our service name being `prison_visits`)
- `23791776` = `Browser` text field